### PR TITLE
Rename EVENTCHKINF flags for payments to Happy Mask salesman

### PR DESCRIPTION
**Pull request by JordanLongstaff**
Tarred the flags for all four mask fees being paid back to the Happy Mask salesman with the same brush.

### CODE REVIEW

File: include/z64save.h [hunk 1]
```diff
@@ -559,10 +559,10 @@ typedef enum LinkAge {
...
 #define EVENTCHKINF_80 0x80
 #define EVENTCHKINF_82 0x82
-#define EVENTCHKINF_8C 0x8C
-#define EVENTCHKINF_8D 0x8D
-#define EVENTCHKINF_8E 0x8E
-#define EVENTCHKINF_8F 0x8F
+#define EVENTCHKINF_PAID_BACK_KEATON_MASK_FEE 0x8C
+#define EVENTCHKINF_PAID_BACK_SKULL_MASK_FEE 0x8D
+#define EVENTCHKINF_PAID_BACK_SPOOKY_MASK_FEE 0x8E
+#define EVENTCHKINF_PAID_BACK_BUNNY_HOOD_FEE 0x8F
```
Comment (fig02) [inline include/z64save.h:565]: personally find the word "fee" unnecessary and would like if the flag ended on the mask name

Comment (JordanLongstaff) [reply include/z64save.h:565]: I'd like to make sure the name captures the fact that it's our payment back to the Happy Mask salesman that happened. So maybe something like `EVENTCHKINF_PAYMENT_RECEIVED_FOR_???_MASK`.

Comment (JordanLongstaff) [reply include/z64save.h:565]: Or `EVENTCHKINF_PAID_BACK_FOR_???_MASK`.

Comment (JordanLongstaff) [reply include/z64save.h:565]: Or `EVENTCHKINF_REIMBURSED_FOR_???_MASK`.

Comment (fig02) [reply include/z64save.h:565]: I still think that "paid back x mask" is fine personally. There is no other person or thing you can pay back for masks in the game, its all pretty clear from context. But sure I guess `EVENTCHKINF_PAID_BACK_FOR_???_MASK` is fine if it makes a big difference for you.
`EVENTCHKINF_REIMBURSED_FOR_???_MASK` sounds like you as the player are getting reimbursed for the expense, which is not what happens

**Review by fig02 — APPROVED** [review]

**Review by cadmic — APPROVED** [review]

**Pull request merged**